### PR TITLE
The test was expecting a  in the closure. Which was wrong. It should be ...

### LINKED ISSUE #54: Possible bug with the aura closure view
Hi Paul, 

I was just playing around to learn and implement ADR in aura. See https://github.com/harikt/adr/commit/f7f79e898d182c4fb9e0222044b9d1224e3498ad

In that I was writing a test on the render functionality done with aura/view. I am seeing something like 

``` php
PHPUnit 4.0.13 by Sebastian Bergmann.

Configuration read from /var/www/adr/tests/phpunit.xml

E.

Time: 30 ms, Memory: 3.75Mb

There was 1 error:

1) Aura\Action_Bundle\AuraViewTest::testRenderHello
Missing argument 1 for Aura\View\View::Aura\Action_Bundle\{closure}()

/var/www/adr/tests/src/AuraViewTest.php:17
/var/www/adr/vendor/aura/view/src/View.php:58
/var/www/adr/vendor/aura/view/src/View.php:58
/var/www/adr/vendor/aura/view/src/View.php:32
/var/www/adr/src/AuraView.php:24
/var/www/adr/tests/src/AuraViewTest.php:38

FAILURES!
Tests: 2, Assertions: 1, Errors: 1.
```

This sounds to me that there is some sort of bug in the View.

### PR DESCRIPTION
**Pull request by harikt**
...using ->variable inside the template. Also fixt the autoloading of the tests.

Fixes #54 where I am wrong.

### CODE REVIEW

**Comment by pmjones** [comment]
What is this "engine" stuff? If you can make a test against _just_ Aura.View that will be more helpful.

File: tests/src/AuraView.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Aura\View;
+
+class AuraView
+{
+    protected $engine;
+
+    public function __construct($engine)
+    {
+        $this->engine = $engine;
```
Comment (harikt) [inline tests/src/AuraView.php:10]: @pmjones the engine is `Aura\View\View` I was trying to make a render system injected . See https://github.com/harikt/adr/blob/master/src/AuraView.php#L6

Comment (pmjones) [inline tests/src/AuraView.php:10]: /me nods

I get that -- even so, it's not a test for Aura.View, it's a test to mimic other systems. The Aura.View tests should be for Aura.View.

Comment (harikt) [inline tests/src/AuraView.php:10]: ok. Just created for you asked :) . This was what I was doing in the same test on the other. I will close this.

Comment (pmjones) [inline tests/src/AuraView.php:10]: Ahh, I see now. :-)  I wanted a test to find the problem with Aura.View, but it appears it was with the function params usage. I need to document that. Thanks man :-)

**Comment by harikt** [comment]
Closing this issue for this was not the test you were looking for.

**Pull request closed without merging**